pkg/controller/deadmanssnitchintegration/deadmanssnitchintegration_controller.go

```diff
@@ -361,7 +364,7 @@ func (r *ReconcileDeadmansSnitchIntegration) createSnitch(dmsi *deadmanssnitchv1
 		} else {
```

Comment:
You don't need the else here please remove it

Reply:
unless i'm misreading things, that `else` wasn't part of this commit. should @sam-nguyen7 remove it anyways?

Reply:
It wasn't but seems to be an easy removal :)